### PR TITLE
[redis]: Disable snapshotting completely

### PR DESCRIPTION
**Pull request by stcheng**
Signed-off-by: Shuotian Cheng <shuche@microsoft.com>

### CODE REVIEW

**Comment by stcheng** [comment]
To address issue https://github.com/Azure/SONiC/issues/59

**Comment by lguohan** [comment]
Retest this please

**Comment by lguohan** [comment]
Retest this please

**Comment by lguohan** [comment]
Retest this please

File: dockers/docker-database/Dockerfile.j2 [hunk 1]
```diff
@@ -24,7 +24,10 @@ debs/{{ deb }}{{' '}}
 RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
 RUN rm -rf /debs
 
-RUN sed -ri 's/^daemonize yes$/daemonize no/;                                           \
+RUN sed -ri 's/^save 900 1$/# save 900 1/;                                              \
+             s/^save 300 10$/# save 300 10/;                                            \
+             s/^save 60 10000$/# save 60 10000/;                                        \
```
Comment (lguohan) [inline dockers/docker-database/Dockerfile.j2]:
```
sed 's/^\(save .*$\)/#\1/g'
```

Comment (stcheng) [inline dockers/docker-database/Dockerfile.j2]: ok. with -r no need to have '\' before brackets

**Comment by stcheng** [comment]
@lguohan updated

**Pull request merged**